docs/ca/dev/monitoring/opentelemetry-instrumentation.md

```diff
@@ -0,0 +1,442 @@
+Event will be wired into the root span.
+
+## Error handling
+
+You are definitely want to track all possible errors during the execution. OOTB OpenTelemetry integration will catch all exceptions that were thrown during request/command execution and attach those as events into the root span. Those events should also be present in the span of the method that thrown the exception, but only if we have a hook for this method in the first place.
+In order to catch those error in any case we would recommend to use `\Spryker\Service\Monitoring\MonitoringService::setError()` or `\Spryker\Service\Opentelemetry\OpentelemetryService::setError()` inside of the Error Handler of your application.
+Default Spryker's Error Handler already executes `\Spryker\Service\Monitoring\MonitoringService::setError()`, so if you are using `Monitoring` module and default Error Handler - you are covered. But if you don't, please adjust your error handler accordingly.
+This will add a error event into the root span and will change its status to the `error` one. We would highly recommend to check this part during integration of OpenTelemetry into your system.
+
+## Changing a service name
+
+Service name allows you to filter traces by the source. You may want to check only Yves traces or mb just to check only Glue requests, but you backend can show you a bunch of CLI commands from your Scheduler.
+You can define a service name via `\Spryker\Service\Monitoring\MonitoringService::setApplicationName()` or `\Spryker\Service\Opentelemetry\OpentelemetryService::setResourceName()`.
+If you are using the `MonitoringService` and any method from it was executed, you will get a generated service name in any case. By default it looks like `APPLICATION-REGION_OR_STORE(application.env)`, where `APPLICATION` is a name of your application (ZED, YVES, GLUE, etc.), `REGION_OR_STORE` is a current store or region name that depends if your application works in Dynamic Store mode or not, and `application.env` is an env name from your deploy file. You can change any of those value via `MonitoringService` or change the name completely via `OpentelemetryService`.
+If nothing was provided by services, OpenTelemetry integration will try to resolve a service name for you.
+First of all `OTEL_SERVICE_NAME_MAPPING` value will be fetched and will try to find a proper service name based on the URL or CLI binary file name.
+If no value was provided or nothing from it was suitable or us - the default service name from `OTEL_DEFAULT_SERVICE_NAME` env value will be used.
+
+## Changing a trace name
+
+Trace name (or in our case a root span name) should show you what request or command was executed. By default it should include a HTTP method name with an URL for the WEB requests and command name for a command execution.
```

Comment:
`By default it should include`

So you're not sure about what it includes?

Reply:
well, for CLI command it will not include HTTP things, but yeah the line can be changed